backend/app/routers/auth.py

```
@@ -177,6 +208,9 @@
     if not user or not auth_utils.verify_password(request.password, user.password):
         raise HTTPException(status_code=401, detail=INVALID_CREDENTIALS)
 
+    access_token = create_access_token(
+        data={"sub": user.email}, expires_delta=timedelta(minutes=30)
```

Ruff (0.8.2)

Trailing comma missing

(COM812)